keras/engine/training.py

```diff
@@ -26,6 +26,18 @@
 from ..legacy import interfaces
 
 
+def _is_tf_tensor(x):
```

Comment:
This and everything that uses it is too tf specific for this file. See how I handled these same issues in #6928